### PR TITLE
DOCS-3179 Browser RUM Data Collected Edits + Preview Link

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Addresses copy nits in the Data Collected doc page + pulls two other preview branches.

### Motivation

DOCS-3179

### Preview

https://docs-staging.datadoghq.com/alai97/rum-data-collected-documentation-update-preview/real_user_monitoring/browser/data_collected/

### Additional Notes

Reminder to myself to change the preview branches back to `develop` before merging!

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/browser/data_collected.md
Comment (alai97) [inline content/en/real_user_monitoring/browser/data_collected.md:37]:
```suggestion
| Session   | 30 days   | A user session begins when a user starts browsing the web application. It contains high-level information about the user (browser, device, geo-location). It aggregates all RUM events collected during the user journey with a unique `session.id` attribute. **Note:** The session resets after 15 minutes of inactivity. |
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
No copy edits to make, but two things:
- should probably undo the changes to pull_config_preview before we merge, and
- There's a broken link being reported by the build

File: local/bin/py/build/configurations/pull_config_preview.yaml
Comment (kayayarai) [inline local/bin/py/build/configurations/pull_config_preview.yaml:554]:
```suggestion
      branch: master
```

File: local/bin/py/build/configurations/pull_config_preview.yaml
Comment (kayayarai) [inline local/bin/py/build/configurations/pull_config_preview.yaml:312]:
```suggestion
      branch: alai97/master
```

**Review by kayayarai — APPROVED** [review]
approved if you use these changes

**Pull request merged**
